ddtrace/appsec/rules.json

```diff
@@ -18,6 +18,9 @@
             "inputs": [
               {
                 "address": "server.request.headers.no_cookies"
+              },
+              {
+                "address": "server.request.cookies"
```

Comment:
Did you manually update this file or was it copied from a shared source?

I am mostly curious if we need to actually audit each change or we can assume this file has the correct values?